src/test/java/jenkins/security/plugins/ldap/LDAPConfigurationWithFIPSTest.java

```diff
@@ -0,0 +1,49 @@
+package jenkins.security.plugins.ldap;
+
+import hudson.util.FormValidation;
+import hudson.util.Secret;
+import jenkins.security.FIPS140;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.FlagRule;
+import org.jvnet.hudson.test.JenkinsRule;
+
+/**
+ * Tests {@link LDAPConfiguration} in FIPS mode.
+ */
+public class LDAPConfigurationWithFIPSTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
```

Comment:
Could you use a `RealJenkinsRule` so that we could benefit from https://github.com/jenkinsci/jenkins-test-harness/pull/829?